gsoc/2009.html

```diff
@@ -49,7 +49,7 @@
         <li><a href="/index.html#projects">Projects</a></li>
         <li><a href="/index.html#internship-opportunities">Internship Opportunities</a></li>
         <li><a href="/index.html#welcome-newcomers">Welcome Newcomers</a></li>
-        <li class="toggle"><a href="/index.html#main-menu">About Us <span class="fas fa-chevron-down" "arrow"></i></a>
```

Comment:
I think that ```<i>``` tag should not be there either or the closed ```</i>``` is mistyped and it should be ```</li>``` after ```</a>```. And ```</span>``` is also missing, this ambiguity is in each of the files changed.

Reply:
@simsausaurabh the `</li>` is closed after the toggle list ends, so that is in right place.
https://github.com/prssanna/systers.github.io/blob/39899c81360e8c77e330173c095bdc381c4fc157/gsoc/2009.html#L58
But yes, the `</span> needs to be added.`
@prssanna Please update the `</span>` tags as well

Reply:
@sara-02 thanks for the info, yes only ```</span>``` needs to be inserted. I didn't see the ```</li>``` tag earlier. @prssanna please update the changes, and it will be good to be merged.

Reply:
@prssanna Please also resolve the conflicts.